### DISCUSSION TITLE
Does this mod fix Qlvl's

### DISCUSSION THREAD

**Discussion by FishGaming** [diasurgical/DevilutionX#6279] [Q&A] [answered]
Hello,

I just ran across this amazing mod for Diablo 1. I found through reading that this mod fixes the issue with uniques not dropping or being morphed due to qlvl conflicts, but does this mod fix the prefix/suffix issues with those that have qlvl of 60? Having only Wirt sell them makes it very boring and almost impossible to obtain, but also the fact that Wirt doesn't sell staves so the prefix/suffix for staves that have qlvl's of 60 are impossible to achieve is also part of the bad coding for Hellfire. Has this been modified in this version of the game?

**Comment by DakkJaniels** [comment, marked as answer]
No

**Reply by FishGaming** [reply]
Thank you for the answser! I wish that was something that was modified.

**Reply by julealgon** [reply]
@FishGaming I think you have a strong case on "impossible staff mods" there... perhaps Wirt _should_ sell staves after all?

**Comment by FishGaming** [comment]
Does the game make any changes to the gold limit on Wirt so that he may even provide the highest qlvls on the best bases?

**Reply by AJenbo** [reply]
no

**Reply by julealgon** [reply]
@FishGaming you should consider creating separate discussions to discuss separate topics, or just make your topic more open here. Also note that this is not a mod 😉 .

**Reply by FishGaming** [reply]
@julealgon I will do that in just a minute to keep everything more organized.
